actions/class.Runner.php

```diff
@@ -970,6 +1018,35 @@ public function messages()
         $this->returnJson($response, $code);
     }
 
+    private function mustForceRestoreTimersFromClient(
+        QtiRunnerServiceContext $serviceContext,
+        $initResponse
+    ): bool {
+        if (!$this->isPausingConcurrentSessionsEnabled()) {
+            return false;
+        }
+
+        if (isset($initResponse['success']) && $initResponse['success']) {
+            $currentState = $serviceContext->getTestSession()->getState();
+
+            return ($currentState != AssessmentTestSessionState::INITIAL);
+        }
+
+        return false;
```

Comment:
Can be simplified by:

```suggestion
        return $this->isPausingConcurrentSessionsEnabled() 
            && isset($initResponse['success']) 
            && $initResponse['success'] 
            && $serviceContext->getTestSession()->getState() !== AssessmentTestSessionState::INITIAL;
```